src/main.rs

```diff
@@ -142,30 +152,46 @@ impl FuzzProject {
     fn exec_target<'a>(&self, args: &ArgMatches<'a>) -> Result<()> {
         let target: String = args.value_of_os("TARGET").expect("TARGET is required").to_os_string()
             .into_string().map_err(|_| "TARGET must be valid unicode")?;
+        let release: bool = args.is_present("release");
+        let assertions: bool = args.is_present("debug_assertions");
         let exec_args = args.values_of_os("ARGS");
         let target_triple = "x86_64-unknown-linux-gnu";
 
         env::set_current_dir(self.path())?;
-        let mut flags = env::var("RUSTFLAGS").unwrap_or("".into());
-        if !flags.is_empty() {
-            flags.push(' ');
-        }
-        flags.push_str("-Cpasses=sancov -Cllvm-args=-sanitizer-coverage-level=3 -Zsanitizer=address -Cpanic=abort");
+
+        let other_flags = env::var("RUSTFLAGS").unwrap_or("".into());
+        let mut flags: Vec<&str> = vec![
+            "-Cdebug-assertions",
+            "-Cpasses=sancov",
+            "-Cllvm-args=-sanitizer-coverage-level=3",
+            "-Zsanitizer=address",
+            "-Cpanic=abort",
+        ];
+        flags.extend(other_flags.split(" "));
+        let rustflags = if assertions {
+            &flags[..]
+        } else {
+            &flags[1..]
```

Comment:
This slicing logic seems like an easy way for a bug to creep up in the future. Why not just `insert` or `extend` the `Vec` instead?